tests/framework/helpers/HtmlTest.php

```diff
@@ -1390,7 +1397,7 @@ public function testError()
      */
     public function testCollectError()
     {
-        $model = new DynamicModel(compact('attr1', 'attr2'));
+        $model = new DynamicModel(['attr1', 'attr2']);
```

Comment:
@rob006 I agree that this code is not equivalent but I don't understand why we should use compact() here. We just want to set model attributes, nothing more. Am I wrong?

Reply:
I'm not sure, I don't understand intention of this test either, and because of that I'm not sure that this change is correct. Maybe the point was to test adding errors to undefined attributes?

Reply:
I think it's about error duplication in summary info and it's work right way.

Reply:
Indeed `compact` here does nothing and it can be skipped or replaced with empty array. The purpose of this test it to check error messages and you can add error messages to non-existing attributes.

I guess author if this test didn't know about `compact` behavior or left it as a hint for attributes used in test.

Reply:
Compact here is, obviously, a mistake.